src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/ElasticSuite.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Process\Deploy\InstallUpdate\ConfigUpdate\SearchEngine;
+
+use Magento\MagentoCloud\Config\ConfigMerger;
+use Magento\MagentoCloud\Config\Environment;
+use Magento\MagentoCloud\Config\Stage\DeployInterface;
+use Magento\MagentoCloud\Package\Manager;
+
+/**
+ * Returns version of elasticsearch
+ */
+class ElasticSuite
+{
+    const NUMBER_OF_SHARDS = 3;
```

Comment:
why do you set these default values? Default values should be common for all environment types.

Reply:
Fixed